chromium_src/BUILD.gn

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2018 GitHub, Inc.
+# Use of this source code is governed by the MIT license that can be
+# found in the LICENSE file.
+
+# Builds some of the chrome sources that Electron depends
+# on unconditionally.
+source_set("chrome") {
+  visibility = [ "//electron:electron_lib" ]
+  sources = [
+    "//chrome/browser/ssl/security_state_tab_helper.cc",
+    "//chrome/browser/ssl/security_state_tab_helper.h",
```

Comment:
should the devtools sources from brightray also go here?

Reply:
The brightray devtools sources aren't used outside of it, when we have moved the relevant `inspectable_web_contents_*` into electron, I am hoping to move the rest here. As for `SecurityStateTabHelper` its used by `atom/browser/common_web_contents_delegate` which acts as the delegate for devtools web contents, and relevant security tab info strings are obtained through it.